package.json

```diff
@@ -9,9 +9,8 @@
         "test": "jest",
         "styleguide": "styleguidist server",
         "styleguide:build": "styleguidist build",
-        "clean": "rimraf public/build/admin",
-        "build": "npm run clean && webpack -p",
-        "watch": "npm run clean && webpack -d -w"
+        "build": "webpack -p",
+        "watch": "webpack -d -w"
```

Comment:
This change would also have to go into `sulu-minimal`, right? Otherwise the build is cleaned twice.

Reply:
done: https://github.com/sulu/sulu-minimal/pull/141